### PR TITLE
Add ultimate vocal remover gui

### PR DESCRIPTION
**Pull request by hamlynt**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [X] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [X] `brew audit --new-cask <cask>` worked successfully.
- [X] `brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @hamlynt.

Could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by hamlynt** [comment]
`createzap` reports no settings found, after starting and closing the app, and after searching multiple combinations of the words in the app name. A manual check of `~/Library/Preferences` and `~/Library/Application Support` also doesn't turn up anything from the app.

File: Casks/ultimate-vocal-remover.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "ultimate-vocal-remover" do
+  arch arm: "arm64", intel: "x86_64"
+
+  version "5.5.0"
+  sha256 arm:   "51f7927c04dea12e9e8acee9326eb195c5532c5142437633b0fbcaa92c5f4613",
+         intel: "375c79e58cb1f7d8def6749b8f7a0ec3648e15c8743f0456a4523ed92f202682"
+
+  url "https://github.com/Anjok07/ultimatevocalremovergui/releases/download/v#{version}/Ultimate_Vocal_Remover_v#{version.major_minor.dots_to_underscores}_MacOS_#{arch}.dmg"
+  name "Ultimate Vocal Remover"
+  desc "Uses state-of-the-art source separation models to remove vocals from audio files"
```
Comment (bevanjkay) [inline Casks/ultimate-vocal-remover.rb:10]: @hamlynt Can you please have a look at update the `desc` stanza?
https://docs.brew.sh/Cask-Cookbook#stanza-desc
```
Do not use empty marketing jargon.
```

**Review by bevanjkay — DISMISSED** [review]

File: Casks/ultimate-vocal-remover.rb (hunk 1 above)
Comment (hamlynt) [reply Casks/ultimate-vocal-remover.rb]: Agreed and revised

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @hamlynt.

**Pull request merged**